release-notes/3.0/preview/api-diff/preview8/.Net/3.0-preview8_System.Collections.Generic.md

```diff
@@ -0,0 +1,11 @@
+# System.Collections.Generic
+
+``` diff
+ namespace System.Collections.Generic {
+     public class SortedList<TKey, TValue> : ICollection, ICollection<KeyValuePair<TKey, TValue>>, IDictionary, IDictionary<TKey, TValue>, IEnumerable, IEnumerable<KeyValuePair<TKey, TValue>>, IReadOnlyCollection<KeyValuePair<TKey, TValue>>, IReadOnlyDictionary<TKey, TValue> {
+-        void System.Collections.ICollection.CopyTo(Array array, int index);
++        void System.Collections.ICollection.CopyTo(Array array, int arrayIndex);
```

Comment:
I see this was changed here, but why?
https://github.com/dotnet/corefx/pull/39305

Most of the other `System.Collections.ICollection.CopyTo` APIs have index as the parameter name and so does the interface itself (except for `Stack<T>`).
https://github.com/dotnet/corefx/blob/9ac414a6781d05ce7d1d26488a72e6b8201bb071/src/Common/src/CoreLib/System/Collections/ICollection.cs#L17
https://docs.microsoft.com/en-us/dotnet/api/system.collections.icollection.copyto?view=netframework-4.8#System_Collections_ICollection_CopyTo_System_Array_System_Int32_
https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.sortedlist-2.system-collections-icollection-copyto?view=netcore-

I am asking here to double check (and not proposing we change anything within the API ref itself). I suppose reverting this change back to what it was in 2.1 is fine (and that's what the source has). Looks like this is reverting a change made in preview 7:
https://github.com/dotnet/core/blob/master/release-notes/3.0/preview/api-diff/preview7/3.0-preview7_System.Collections.Generic.md

Reply:
@wtgodbe don't we do API Compat runs for this sort of stuff? We might only compare against the last stable release, but argubably we should include previews to make sure that we don't have unintentional breaking changes (intentional ones would be recorded as a baseline'd violation. @stephentoub, any thoughts on this?